README.md

```diff
@@ -1,6 +1,12 @@
 # Sort Package.json
 
-[![Build Status](https://travis-ci.org/keithamus/sort-package-json.svg)](https://travis-ci.org/keithamus/sort-package-json)
+[![Build Status][github_actions_badge]][github_actions_link]
+[![NPM Version][package_version_badge]][package_link]
+
+[package_version_badge]: https://img.shields.io/npm/v/sort-package-json.svg
+[package_link]: https://www.npmjs.com/package/sort-package-json
+[github_actions_badge]: https://img.shields.io/github/actions/workflow/status/keithamus/sort-package-json/pr.yml
+[github_actions_link]: https://github.com/keithamus/sort-package-json/actions?query=workflow%3ACI+branch%3Amaster
```

Comment:
@keithamus Since I'm working on this, do you think it's a good opportunity to rename `master` to `main`? Unless you don't want to.

Reply:
Yes please.

Reply:
Done, https://github.com/keithamus/sort-package-json/pull/288/commits/4270583013242d13081665b6a65d10e08fe563f7.

Also made some changes to CI config.

I don't have access to rename branch, you'll have to do it.

Reply:
I've renamed to `main` :+1:

Reply:
Good to go.